src/lib/Components/Bidding/Components/Button.tsx

```diff
@@ -1,13 +1,18 @@
 import React from "react"
-import { StyleSheet } from "react-native"
+import { StyleSheet, View } from "react-native"
 
-import { ButtonProps, GhostButton, InvertedButton } from "../../Buttons"
+import InvertedButton from "lib/Components/Buttons/InvertedButton"
+import { ButtonProps, GhostButton } from "../../Buttons"
 
 export class Button extends React.Component<ButtonProps> {
   render() {
     const { style, textStyle, ...props } = this.props
 
-    return <InvertedButton style={[styles.default, style]} textStyle={[styles.text, textStyle]} {...props} />
+    return (
+      <View style={[styles.default, style]}>
```

Comment:
@yuki24 how come we're not using a styled component here?

Reply:
We can totally use it here! will update it.